sdk/keyvault/azure-security-keyvault-keys/test/ut/key_client_base_test.hpp

```diff
@@ -33,6 +33,16 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys { nam
     int m_testPollingTimeOutMinutes = 20;
     std::chrono::milliseconds m_testPollingIntervalMs = std::chrono::minutes(1);
 
+    void CheckSkipHSMForLive()
+    {
+      // if we are in live mode and the test needs a valid HSM url (aka different from the keyvault
+      // url)
+      if (m_testContext.IsLiveMode() && (m_keyVaultHsmUrl != m_keyVaultHsmUrl))
```

Comment:
`m_keyVaultHsmUrl != m_keyVaultHsmUrl`?!
Not equal to itself?

Reply:
i'm stupid, i just woke up and not had my coffee yet